### ISSUE TITLE
A lot of stuff isn't either mono or at 44100 so it doesn't work

### ISSUE THREAD

**Issue by AgentWhatever** [Metastruct/garrysmod-chatsounds#50]
Get somebody/something to resample everything?

**Comment by PotcFdk** [comment]
The preprocessor should be checking this and print warnings.
Can you give me a sample list of sounds that don't work ingame, so I can adjust the checks?

**Comment by AgentWhatever** [comment]
http://puu.sh/mA9U7/c5f14b0d3a.png
To list just a few

**Comment by PotcFdk** [comment]
Sorry, what?
```
\sound\chatsounds\autoadd\markorepairs\but>ffmpeg -i but1.ogg
[...]
Input #0, ogg, from 'but1.ogg':
  Duration: 00:00:00.92, start: 0.000000, bitrate: 93 kb/s
    Stream #0:0: Audio: vorbis, 44100 Hz, mono, fltp, 80 kb/s
    Metadata:
      ENCODER         : Lavc55.71.100 libvorbis
```

Plays fine in my GMod: `surface.PlaySound("chatsounds/autoadd/markorepairs/but/but1.ogg")`
Are you up-to-date?
Can you check the checksum of that file?

```
  File: but1.ogg
CRC-32: 8d73abd5
   MD4: 711aed135c6a6dc45966a9e72d67e643
   MD5: 2ef35abfb4b419df2488702d4fd4a3d1
 SHA-1: 70fe7592d8fdf0b48f71180f61c804daecccb209
```

**Comment by AgentWhatever** [comment]
I am.
MD5 : 2EF35ABFB4B419DF2488702D4FD4A3D1
SHA-1: 70FE7592D8FDF0B48F71180F61C804DAECCCB209

on \sound\chatsounds\autoadd\markorepairs\but\but1.ogg

**Comment by PotcFdk** [comment]
@Python1320 any idea what could cause his client to fail to play the file?

**Comment by Techbot121** [comment]
it might be your own audio sample rate you're using as playback

**Comment by AgentWhatever** [comment]
24bit, 192000 Hz

**Comment by PotcFdk** [comment]
Python said it might be Garry's Mod's fault.

**Comment by TrenchFroast** [comment]
bad bad, not being mono is not the problem

**Comment by LUModder** [comment]
I'm probably partially to blame for that

**Comment by Tenrys** [comment]
I remember having a problem with chatsounds once, sounds wouldn't play at all and be considered as if they weren't in the required frequency, and it didn't stop happening until I reinstalled my system.

**Comment by PotcFdk** [comment]
See also https://github.com/Facepunch/garrysmod-issues/issues/1430

**Comment by DaftLad** [comment]
I used to have this exact same problem with Chatsounds. It wouldn't matter what format it was, it'd throw an error at me saying it wasn't in the correct format.
I've heard of two solutions; reinstall every single Source game to get rid of any remnants from older versions of Source (how the hell are you supposed to know that anyway?), or move Garry's Mod to a different harddrive. I've tried both of them with no avail.
I'm surprised this hasn't been solved yet, is this issue **that** uncommon?
